spec/models/group_spec.rb

```diff
@@ -18,8 +18,9 @@ def new_user
 
   it "is invalid without a name" do
     group = Group.new()
-    expect(group).not_to be_valid
-    expect(group.errors[:name]).to include "can't be blank"
+    expect(Group.count).to eq 0
+    # expect(group).not_to be_valid
+    # expect(group.errors[:name]).to include "can't be blank"
```

Comment:
This test doesn't work without the expectations as new never saves in the db. Best uncomment these lines again